### ISSUE TITLE
Small mistake in help for GaudiExec?

### ISSUE THREAD

**Issue by KonstantinSchubert** [ganga-devs/ganga#749]
This assignment failed when I tried it: 
```
myApp.options = "$SOMEPATH/DaVinciDev_v40r2/myDaVinciOpts.py"
```
https://github.com/ganga-devs/ganga/blob/cd86b6eb19fa652b276c1d242c6dccce08db647c/python/GangaLHCb/Lib/Applications/GaudiExec.py#L66

Instead, I had to assign a list:

```
myApp.options = ["$SOMEPATH/DaVinciDev_v40r2/myDaVinciOpts.py"]
```

**Comment by rob-c** [comment]
@KonstantinSchubert Ah yes, I forgot to update the docs when I made the changes to support lists.
Shall I make a PR for this or do you want to? It may be worth checking that there aren't any other typos I think, thanks for reporting!

**Comment by KonstantinSchubert** [comment]
I will make a PR, become a contributor :D ;)

**Comment by KonstantinSchubert** [comment]
Zu welcher Branch mache ich den PR? Develop?

**Comment by rob-c** [comment]
If you can make the PR against develop it might be in time to get in before 6.1.25.
@mesmith75 Have you started the release process yet or can we get this in?

**Comment by mesmith75** [comment]
I am afraid the release has already been started.

**Comment by alexanderrichards** [comment]
I think one could equally argue since it's a typo it can wait to 6.2.1. I don't mind either way, it's your call as your release manager however if you make changes to the release branch then they will need to be merged back to develop once the release is done. As long as this is done then I don't thing there will be a problem

**Comment by milliams** [comment]
All changes to the release branch are automatically merged back into
develop when the release is finalised.

Cheers,
Matt

**Comment by alexanderrichards** [comment]
ah ha magic! :wink:

**Comment by mesmith75** [comment]
Yes let's put it in the release. It is a new(ish) feature so the helper text should probably be accurate.

**Comment by mesmith75** [comment]
@KonstantinSchubert Can you make a new pull request against the release/6.1.25 branch. Then I can finish the release.

**Comment by milliams** [comment]
It's now (in the last few weeks or so) possible to change the branch against which a PR is made.

**Comment by mesmith75** [comment]
So it is. Thanks!

**Comment by KonstantinSchubert** [comment]
Sure, give me a minute.

Mark Smith <notifications@github.com> schrieb am Mi., 24. Aug. 2016, 13:03:

**Comment by rob-c** [comment]
Closed via #750
